packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -1854,7 +1854,7 @@ export class EditableGrid extends PureComponent<EditableGridProps, EditableGridS
         }
 
         return (
-            <div>
+            <div className="editable-grid-wrapper">
```

Comment:
I didn't see any css related to this class name, was it added for selenium test locator or something along those lines?

Reply:
It's needed by the ui-premium PR that I am working on which adds some styles for this specific div in order to get the scrolling behavior correct. I'm not done with that PR quite yet though. But also, in general, bare divs are unhelpful. It's almost impossible to tell who is generating them (it was tricky in this case for me), and they make it harder for us to have specific enough selectors in our selenium tests.